airflow/api_connexion/schemas/dag_run_schema.py

```diff
@@ -109,9 +101,8 @@ def autogenerate(self, data, **kwargs):
 
     @post_dump
     def autofill(self, data, **kwargs):
-        """Populate execution_date from logical_date for compatibility."""
```

Comment:
Same here. I suspect this entire function can be removed?

Reply:
Changed the documentation as it autofills other fields as well